### PR TITLE
Ofri oi climber placer

### PR DESCRIPTION
**Pull request by OfriRosenbaum**

### CODE REVIEW

File: src/main/java/frc/robot/commands/DropBothPlacers.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.ClimberPlacer;
+
+public class DropBothPlacers extends ParallelCommandGroup {
+
+    public DropBothPlacers() {
+        ClimberPlacer leftPlacer = ClimberPlacer.getLeftInstance();
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/DropBothPlacers.java:9]: unbelievably unnecessary but fineeeee

File: src/main/java/frc/robot/commands/DropBothPlacers.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        ClimberPlacer rightPlacer = ClimberPlacer.getRightInstance();
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/DropBothPlacers.java:10]: image

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/OI.java [hunk 3]
```diff
@@ -10,40 +11,59 @@
...
+    private static final Joystick left = new Joystick(0);
+    private static final Joystick right = new Joystick(1);
+    private static final XboxControllerWrapper xbox = new XboxControllerWrapper(2);
 
     /**
-     * <b>Trigger of Left Joystick</b>: Aim To Cargo<br>
+     * <b>Trigger of Left Joystick</b>: Reverse transfer<br>
+     * <b>Trigger of Right Joystick</b>: Release cargo<br>
      * <b>Right Trigger</b>: Intake cargo<br>
      * <b>Right Bumper</b>: Intake placer up<br>
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/OI.java:22]: change to `IntakePlacer up`

Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/OI.java:22]: change to `IntakePlacer up`

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/OI.java [hunk 4]
```diff
@@ -52,10 +72,18 @@ public OI() {
...
 
         //reverse the roller and intakeToTransfer to return the cargo that is in the intakeToTransfer
-        xbox.getLeftButton().whileHeld(new ParallelCommandGroup(
+        xbox.getUpButton().whileHeld(new ParallelCommandGroup(
                 new MoveGenericSubsystem(roller, IntakeRoller.MAX_SPEED),
                 new MoveGenericSubsystem(intakeToTransfer, -IntakeToTransfer.SPEED)
         ));
+
+        xbox.getLeftButton().whileHeld(new ParallelCommandGroup(
+                new MoveGenericSubsystem(leftPlacer, ClimberPlacer.MIN_SPEED),
```
Comment (tal-sitton) [inline src/main/java/frc/robot/OI.java:81]: image

File: src/main/java/frc/robot/subsystems/ClimberPlacer.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/subsystems/ClimberPlacer.java:1]: remove

File: src/main/java/frc/robot/commands/MoveToNextBar.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/MoveToNextBar.java:1]: remove

File: src/main/java/frc/robot/commands/IntakeCargo.java [hunk 7]
```diff
@@ -38,8 +38,7 @@ public boolean isFinished() {
                                 return (!hasCargo && transfer.getEntranceSensor()) || (hasCargo && intakeToTransfer.getLimit());
                             }
                         }
-                )
-        );
+                ));
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/IntakeCargo.java]: why?

File: src/main/java/frc/robot/commands/DropPlacer.java [hunk 8]
```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/DropPlacer.java:1]: remove

File: src/main/java/frc/robot/commands/DropBothPlacers.java [hunk 9]
```diff
@@ -0,0 +1,14 @@
+package frc.robot.commands;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/DropBothPlacers.java:1]: remove

File: src/main/java/frc/robot/RobotMap.java [hunk 10]
```diff
@@ -21,6 +21,9 @@
```
Comment (tal-sitton) [inline src/main/java/frc/robot/RobotMap.java:21]: why so many new ports

File: src/main/java/frc/robot/Robot.java [hunk 11]
```diff
@@ -8,6 +8,7 @@
 import com.spikes2212.dashboard.RootNamespace;
```
Comment (tal-sitton) [inline src/main/java/frc/robot/Robot.java:8]: why so many new things

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Pull request closed without merging**